scripts/sonar/sonar-controller.js

```diff
@@ -41,23 +50,51 @@ angular.module('app.sonar').controller('SonarCtrl', ['$scope', '$log', '$q', 'ap
                 var totalLines = sum(timePeriod.metrics, 'lines');
 
                 var weightedMetrics = timePeriod.metrics.map(function(project) {
-                    var factor = project.lines / totalLines;
 
                     return {
-                        methodComplexity: factor * project.methodComplexity,
-                        fileComplexity: factor * project.fileComplexity,
-                        coverage: factor * project.coverage,
-                        compliance: factor * project.compliance,
+                        numberOfMethodsInProjectInPeriod: project.totalMethods,
+                        numberOfFilesInProjectInPeriod: project.totalFiles,
+                        complexityInProjectInPeriod: project.totalComplexity,
+                        testableLinesInProjectInPeriod: project.testableLines,
+                        untestedLinesInProjectInPeriod: project.untestedLines,
+
```

Comment:
Probably best not to have a blank line in the middle of an object initialisation, it looks a bit odd.